site-content/source/modules/ROOT/pages/blog/Apache-Cassandra-4.1-Features-Guardrails-Framework.adoc

```diff
@@ -70,7 +70,7 @@ Warnings :
 Guardrail tables violated: Creating table t6, current number of tables 6 exceeds warning threshold of 5.
 ```
 
-However, if the hard limit is reached, the user operation will be aborted with a `GuardrailViolatedException`, preventing the potentially harmful operation from happening. Continuing with the previous example, if we have a hard limit of ten tables (`tables_warn_threshold: 10`) and we try to create an eleventh table, we will see an error and the eleventh table won’t be created:
+However, if the hard limit is reached, the user operation will be aborted with a `GuardrailViolatedException`, preventing the potentially harmful operation from happening. Continuing with the previous example, if we have a hard limit of ten tables (`tables_fail_threshold: 10`) and we try to create an eleventh table, we will see an error and the eleventh table won’t be created:
```

Comment:
Do we regularly sneak things in to PR's for website changes? For the record: definitely not against this. Just want to double check this isn't accidentally flying under the radar here and colliding with other work in flight and/or accidentally pulled over from a larger change.

Reply:
This seems to be the fix to https://issues.apache.org/jira/browse/CASSANDRA-17936 which was indeed merged a few days ago already.